client/landing/stepper/index.tsx

```diff
@@ -92,7 +92,7 @@ async function main() {
 	const flowName = getFlowFromURL();
 	const flowLoader = availableFlows[ flowName ];
 
-	if ( ! flowLoader ) {
+	if ( ! flowLoader || typeof flowLoader !== 'function' ) {
```

Comment:
The `typeof` check should be enough and imply the nullish check too

```suggestion
	if ( typeof flowLoader !== 'function' ) {
```